iotile_ext_cloud/iotile/cloud/utilities.py

```diff
@@ -29,3 +30,14 @@ def device_slug_to_id(slug):
         return int(short, 16)
     except ValueError as exc:
         raise ArgumentError("Invalid device slug with non-numeric components", error_mesage=str(exc), slug=slug)
+
+def get_timestamp(utcdate):
```

Comment:
This custom function isn't needed since we already have a dependency on `python-dateutil` that has a standard way of doing this conversion:

```
In [1]: import dateutil.parser

In [2]: dateutil.parser.parse('Wed, 01 Sep 2010 17:30:32 GMT')
Out[2]: datetime.datetime(2010, 9, 1, 17, 30, 32, tzinfo=tzutc())
```